packages/ember-template-compiler/lib/system/compile.ts

```diff
@@ -1,7 +1,6 @@
 /**
 @module ember
 */
-import require, { has } from 'require';
```

Comment:
Hmm, why wouldn't we want this `require` to be imported? Without this import we are effectively using the global `require` (ala cjs)